devel/xcode-build-server/Portfile

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        SolaWing xcode-build-server 1.0.0 v
+version             1.0.0
+revision            0
+license             MIT
+categories          devel python
+maintainers         {woolsweater.net:macports @woolsweater} \
+                    openmaintainer
+supported_archs     noarch
+
+description         Build Server Protocol implementation for Xcode projects
+
+long_description    ${name} integrates Xcode with Apple's sourcekit-lsp. \
+                    sourcekit-lsp doesn't itself support Xcode projects, but it does \
+                    provide a Build Server Protocol client to work with other \
+                    build systems. xcode-build-server implements a build \
+                    server to provide sourcekit-lsp with the build \
+                    information it needs for an Xcode project.
+
+checksums           rmd160  fd21bf312b47c15fc28d03b80de1ec39d76140a2 \
+                    sha256  08ea2b9a892670bb7c2f8ff90a97ae0be519744f42ba713bb1a56e247c77e885 \
+                    size    18673
+
+build {}
+
+post-patch {
+    reinplace "s|#!/usr/bin/env python3|#!/usr/bin/env python${python.branch}|" \
+        ${worksrcpath}/${name}
+}
+
+destroot {
+    set pkgdir [string replace ${python.pkgd} 0 [string length ${prefix}]]/${name}
+    set destpkgdir ${destroot}${prefix}/${pkgdir}
```

Comment:
creating two new variables here seem unnecessary. I can take a look later tonight when I am behind a computer; this should be doable with variables that are already defined in the "python" PortGroup.

Reply:
I was unable to find a suitable one but I would be happy to take your suggestions. 👍

Reply:
the destroot section should become:
```
destroot {
    xinstall -d ${destroot}/${python.pkgd}/${name}
    xinstall -m 0755 {*}[glob ${worksrcpath}/*.py] ${destroot}/${python.pkgd}/${name}
    xinstall -m 0755 -W ${worksrcpath} ${name} ${destroot}/${python.pkgd}/${name}
    ln -s ${python.pkgd}/${name}/${name} ${destroot}${prefix}/bin/${name}
}
```

Reply:
Done. So that I understand for next time, would you mind letting me know why repeating the literal `${destroot}/${python.pkgd}/${name}` is preferable to making a variable for it and using that?